realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/SyncedRealmTests.java

```diff
@@ -63,7 +63,7 @@ public void waitForInitialRemoteData_mainThreadThrows() {
         final SyncUser user = loginUser();
 
         SyncConfiguration config = new SyncConfiguration.Builder(user, Constants.USER_REALM)
-                .waitForInitialRemoteData()
+                .waitForInitialRemoteData(false)
```

Comment:
I must admit that `false` to me carries little information. Maybe a constant with `READ_ONLY` in the name could increase the readability.